key_teleop/test/test_xmllint.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
```

Comment:
is there a psosibility to share this garbage between the multiple packages in this repo or we have to maintain several copies?

Reply:
I'm not sure if tests are also tested actually. For other files, yup it is necessary.